modules/programs/helix.nix

```
@@ -161,7 +168,22 @@ in {
   };
 
   config = mkIf cfg.enable {
-    home.packages = [ cfg.package ];
+    home.packages = if cfg.extraPackages != [ ] then
+      [
+        (pkgs.symlinkJoin {
+          name = "helix-wrapped-${lib.getVersion pkgs.helix}";
+          paths = [ pkgs.helix ];
+          preferLocalBuild = true;
+          nativeBuildInputs = [ pkgs.makeWrapper ];
+          postBuild = ''
+            rm $out/bin/hx
+            makeWrapper "${cfg.package}/bin/hx" $out/bin/hx \
+              --prefix PATH : ${lib.makeBinPath cfg.extraPackages}
+          '';
```

You can use `wrapProgram` instead:

```suggestion
          postBuild = ''
            wrapProgram $out/bin/hx \
              --prefix PATH : ${lib.makeBinPath extraPackages}
          '';
```